config/locales/server.zh_CN.yml

```diff
@@ -45,7 +45,7 @@ zh_CN:
         topic_not_found_error: "当回复到已经删除的相关主题时发生。"
         topic_closed_error: "当回复到已经关闭的相关主题时发生。"
   errors: &errors
-    format: '%{attribute} %{message}'
+    format: '%{attribute}%{message}'
```

Comment:
Yes, these changes will be overwritten when we pull the latest translations from Transifex.